README.md

```diff
@@ -7,6 +7,28 @@ bells and whistles.
 For a longer description and available features and options, see the
 [man page](mkosi.md).
 
+# Installation
+
+Installing `mkosi` is easy, as it has no runtime dependencies.
+
+If you just want the current master branch you can run
+```shell
+python3 -m pip install --user git+https://github.com/systemd/mkosi.git
+```
+
+If you want to hack on mkosi do
+```shell
+# clone either this repository or your fork of it
+git clone https://github.com/systemd/mkosi/
+cd mkosi
+python3 -m pip install --user --editable .
+```
+This will install mkosi in editable mode to `~/.local/bin/mkosi`, allowing you
+to use your own changes right away.
+
+For development you will also need [mypy](https://github.com/python/mypy) and
```

Comment:
I'd mention this is optional. I actually haven' t bothered to setup mypy locally yet. CI catches everything anyway so I don't mind developers not using pytest and mypy locally.